### ISSUE TITLE
EDU18 Session Title

### ISSUE THREAD

**Issue by profmikegreene** [profmikegreene/conference-proposals#1]
https://github.com/profmikegreene/conference-proposals/blob/working/educause-2018.md#session-title 

"Building a student-first NGDLE that embraces faculty and department choice." That came to mind as something we're trying to achieve with learn.duke is an enterprise system that allows faculty/department/schools to add/select apps that they manage. 

Something I've really started digging the past couple years is the idea of meaningful choice for faculty inside enterprise IT which I think will resonate with the edu18 audience. Hibbett's setup with git/canvas/gravcms  is a major inspiration https://about.gitlab.com/2017/10/12/collaborative-course-environment-gitlab-grav/

@btopro does anything you want to talk about fit that vibe?

**Comment by btopro** [comment]
absolutely. This submission I have in for Apereo is a similar vibe -- https://btopro.com/blog/ngdle-oer-cost-reduction-accessibility-and-decentralization-the-future-of-our-institutions

TLDR; if we embrace faculty choice / decentralization and meet them with tools where they are we can better serve them, produce better outcomes and have less expensive options by aligning with the structure of our organization, not our organizational IT. Focus is more C-level / directors / managers then I tend to do but I'd be all about that title you listed @profmikegreene or something very similar in nature.
